ui/components/multichain/nft-item/nft-item.js

```diff
@@ -58,7 +62,7 @@ export const NftItem = ({
           />
         }
       >
-        {src ? (
+        {isIpfsEnabled ? (
```

Comment:
When the IPFS gateway toggle is off and `isIpfsEnabled` is falsy, this line incorrectly prevents displaying non-IPFS media too.

Check out this example:

<img width="1454" alt="Screenshot 2023-08-01 at 09 35 07" src="https://github.com/MetaMask/metamask-extension/assets/13814744/2ce4e483-e8f1-46eb-ab2f-bf62e0c19055">

Reply:
Good shout! Yeah, I missed this edge case when the image was stored somewhere other than IPFS. Handled it here https://github.com/MetaMask/metamask-extension/pull/20172/commits/df1ccda81462ae23a0229ec67e5f26f81a161771